app/views/api/v1/passwords/update.json.jbuilder

```diff
@@ -0,0 +1,3 @@
+json.success true
+json.user user, partial: 'api/v1/users/info', as: :user
+json.message 'Your password has been successfully updated.'
```

Comment:
I think this and the `success` key are not necessary.
You're rendering this view because was a success, the client receives 200 so it's not necessary to explicitly mention the "success true". Same for the message. The client can show a message if they want.